platform/platform-filter-delegate/src/main/java/org/codice/ddf/platform/filter/delegate/DelegateServletFilter.java

```diff
@@ -112,6 +102,168 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
     @Override
     public void init(FilterConfig filterConfig) throws ServletException {
         this.filterConfig = filterConfig;
+
+        // now that the DelegateServletFilter has been initialized with a filterConfig, initialize all SecurityFilters using that filterConfig
+        for (ServiceReference<SecurityFilter> securityFilterServiceReference : sortedSecurityFilterServiceReferences) {
+            initFilter(securityFilterServiceReference, filterConfig);
+        }
+
+        filtersHaveBeenInitialized = true;
+    }
+
+    public void addSecurityFilter(
+            final ServiceReference<SecurityFilter> securityFilterServiceReference)
+            throws ServletException {
+        LOGGER.trace("Adding SecurityFilter ranking={} name={}",
+                securityFilterServiceReference.getProperty(Constants.SERVICE_RANKING),
+                ServicePropertiesUtils.getStringProperty(securityFilterServiceReference,
+                        HttpWhiteboardConstants.HTTP_WHITEBOARD_FILTER_NAME));
+
+        if (filtersHaveBeenInitialized) {
```

Comment:
The `SecurityFilter`s need a `ServletContext` when they are `init`ed. The `ServletContext` that we want to to use is the one that comes from the `filterConfig` in `DelegateServletFilter#init(FilterConfig filterConfig)`.